packages/grafana-ui/src/components/Switch/Switch.tsx

```diff
@@ -173,5 +177,10 @@ const getSwitchStyles = stylesFactory((theme: GrafanaTheme2, transparent?: boole
     inlineLabelEnabled: css`
       color: ${theme.colors.text.primary};
     `,
+    invalid: css`
+      input:checked + label {
```

Comment:
We shouldn't only apply this when `input` is checked, it should be possible to be unchecked AND invalid

Reply:
Yes, I know. I haven't touched the code since I asked this on the channel. I would do the same with Checkbox, as RadioButtonGroup always has a default checked, right?

Reply:
Hmm `value` of `RadioButtonGroup` is optional, so it might be possible to start with nothing selected? We should show the invalid state in that case as well anyway

Reply:
Copy that! 👌